app/forms/curation_concerns/forms/work_form.rb

```diff
@@ -44,8 +44,6 @@ class << self
         # 'rights' which only has a single value on the form.
```

Comment:
Can you strike `, but we need to override for 'rights' which only has a single value on the form`?